src/pygama/pargen/dplms_ge_dict.py

```diff
@@ -0,0 +1,575 @@
+"""
+This module is for creating dplms dictionary for ge processing
+"""
+
+from __future__ import annotations
+
+import itertools
+import json
+import logging
+import os
+import pathlib
+import pickle
+import time
+from collections import OrderedDict
+
+import lgdo
+import lgdo.lh5_store as lh5
+import matplotlib.pyplot as plt
+import numpy as np
+from lgdo import Array
+from scipy.signal import convolve, convolve2d
+
+from pygama.math.histogram import get_hist
+from pygama.math.peak_fitting import (
+    extended_gauss_step_pdf,
+    extended_radford_pdf,
+    gauss_step_pdf,
+    radford_pdf,
+)
+from pygama.pargen.cuts import find_pulser_properties, generate_cuts, get_cut_indexes
+from pygama.pargen.dsp_optimize import run_one_dsp
+from pygama.pargen.energy_cal import hpge_find_E_peaks
+from pygama.pargen.energy_optimisation import (
+    event_selection,
+    fom_FWHM,
+    fom_FWHM_with_dt_corr_fit,
+)
+
+log = logging.getLogger(__name__)
+sto = lh5.LH5Store()
+
+
+def dplms_ge_dict(
+    lh5_path: str,
+    raw_fft: lgdo.Table,
+    raw_cal: lgdo.Table,
+    dsp_config: dict,
+    par_dsp: dict,
+    par_dsp_lh5: str,
+    dplms_dict: dict,
+    decay_const: float = 0,
+    ene_par: str = "dplmsEmax",
+    display: int = 0,
+) -> dict:
+    """
+    This function calculates the dplms dictionary for HPGe detectors.
+
+    Parameters
+    ----------
+    lh5_path: str
```

Comment:
If you use type annotations in the function signature, there is no need to duplicate them in the docstring:

```suggestion
    lh5_path,
```